inc/tag.class.php

```diff
@@ -702,8 +702,7 @@ public static function showTagDropdown($params = [])
         }
 
         $readOnly = !$tag::canUpdate()
-            || ($obj->isNewItem() && !$obj->canCreateItem())
-            || (!$obj->isNewItem() && !$obj->canUpdateItem())
```

Comment:
What does `$obj->isNewItem()` return? In the case of a massive action, the item already exists, so it should return false. In this case, this is what needs to be fixed.